crates/redis/src/lib.rs

```diff
@@ -50,8 +57,8 @@ impl TriggerExecutor for RedisTrigger {
     fn new(engine: TriggerAppEngine<Self>) -> Result<Self> {
         let address = engine
             .app()
-            .require_metadata("redis_address")
-            .context("Failed to configure Redis trigger")?;
+            .require_metadata::<TriggerMetadata>("trigger")?
+            .address;
```

Comment:
It's only been three weeks and I can't even remember switching back to `trigger` here, let alone _why_. :thinking: 

This fix looks correct so I think its good to merge, but I'll probably be reverting this and fixing the matching metadata generation later (which should be fine as the metadata is an unstable interface).